fvm/evm/emulator/config.go

```diff
@@ -71,7 +71,7 @@ var DefaultChainConfig = &gethParams.ChainConfig{
 	// Fork scheduling based on timestamps
 	ShanghaiTime: &zero, // already on Shanghai
 	CancunTime:   &zero, // already on Cancun
-	PragueTime:   &zero, // already on Prague
+	PragueTime:   nil,   // not on Prague
```

Comment:
Not related to this PR, but we should disable these flag in case we update the Geth version in the future. 
Should not have any impact on the current users as the current version of the underlying Geth code used by this repo doesn't use this flag but it might in the future.